addons/payment_authorize/data/payment_acquirer_data.xml

```diff
@@ -6,6 +6,7 @@
         <field name="inline_form_view_id" ref="inline_form"/>
         <field name="support_authorization">True</field>
         <field name="support_fees_computation">False</field>
+        <field name="support_refund">False</field>
```

Comment:
Authorize.Net has an override of `_send_refund_request`. Doesn't it support refunds? Maybe only full amounts?

Reply:
I'll put `support_partial_refund` instead, it's better.
Seems to me Authorized.net supports only full amounts, but we could maybe test it to be sure. Anyway, the Authorize.net version is used for the refund validation transaction flow, with the full amount.

Reply:
Let's put that aside for now. I feel that we could come up with a simpler solution than two support fields but I don't see which solution that would be just yet.